service_mesh/service_mesh_day_two/ossm-auto-route.adoc

```diff
@@ -0,0 +1,68 @@
+[id="ossm-auto-route"]
+= Automatic route creation
+include::modules/ossm-document-attributes.adoc[]
+toc::[]
+
+OpenShift routes for Istio Gateways are automatically managed in {ProductName}. Every time an Istio Gateway is created, updated or deleted inside the service mesh, an OpenShift route is created, updated or deleted. 
+
+[id="ossm-auto-route-enable_{context}"]
+== Enabling Automatic Route Creation
+A {ProductName} control plane component called Istio Openshift Routing (IOR) synchronizes the gateway route. You can deploy it as part of the control plane in the {ProductName} installation process, but you must enable it first.
+
+. In the `ServiceMeshControlPlane` resource, add the `ior_enabled` parameter and set it to `true`. For example, see the following resource snippet:
+
+----
+gateways:
+ istio-egressgateway:
+   autoscaleEnabled: false
+   autoscaleMin: 1
+   autoscaleMax: 5
+ istio-ingressgateway:
+   autoscaleEnabled: false
+   autoscaleMin: 1
+   autoscaleMax: 5
+   ior_enabled: true
+----
+
+For more information, see xref:../../service_mesh/service_mesh_install/customizing-installation-ossm.adoc#ossm-cr-gateway_customizing-installation-ossm[Istio gateway configuration].
+
+[id="ossm-auto-route-subdomains_{context}"]
+== Subdomains
+
+{ProductName} creates the route with the subdomain, but {product-title} must be configured to enable it. Subdomains, for example, `*.domain.com` are supported but not by default. Cluster administrators can refer to the OpenShift documentation for instructions on how to enable subdomains.
+
+[id="ossm-auto-route-tls_{context}"]
+== TLS
+
+If the Gateway contains a TLS section, the OpenShift Route will be configured to support TLS.
+
+For example, if the following gateway is created:
```

Comment:
This example does not include any TLS section. Not sure if it's on purpose, but the way I read it, I'm expecting the example below to be an example of the paragraph above. If you look at maistra doc, you'll see that this example is in the beginning of the page, it's not tied to TLS.

Reply:
You're right, I was reordering the topic and I moved TLS too high and forgot about it. I moved it back down to the bottom. Thanks!